pegasus/src/database.rb

```diff
@@ -28,6 +28,10 @@ def initialize(table, no_cache: false)
         "#{db_column_name}___#{column_alias}".to_sym
       end
     end
+
+    # Create a new instance of the dynamically generated Sequel Table classes to lazy load the class definition
+    @contents = DB[@table].new
```

Comment:
I'm curious, does just
```suggestion
    DB[@table].new
```
work? I don't love the idea of assigning to `@contents` something that doesn't actually represent the contents, even if just for a single line of code

Reply:
Actually, does just
```suggestion
    DB[@table]
```
work? Glancing through Sequel's codebase, I'd guess that's all that's necessary to get the class definition loaded. The [actual initialization method](https://github.com/jeremyevans/sequel/blob/0d509ad9058edb0c5d43695f55d982fdc2174ec4/lib/sequel/dataset/misc.rb#L17C1-L30) does very little

Reply:
That felt a lot more deliberate to me as well, but I was not sure about potential side effects, more due to my unfamiliarity with Ruby. Validated that just add `DB[@table]` works as well. Updated comment accordingly.